test/pm-plugin-development/plugin-doctor/test_analyze.py

```
@@ -1909,7 +1919,11 @@ def test_unconfident_child_keeps_its_name_but_withdraws_its_flag_set():
 
     assert 'unprobeable' in entry.subcommands, 'unprobeable verb lost its acceptance'
     assert entry.subcommands['unprobeable'] is None, 'unknown flag surface read as empty'
-    assert entry.subcommands['probed'] == {'plan-id', 'field'}
+    # The acceptance surface is root ∪ subtree ∪ the executor/router universals,
```

Ruff (0.16.1)

[warning] Comment contains ambiguous `∪` (UNION). Did you mean `U` (LATIN CAPITAL LETTER U)?

(RUF003)